packages/site/cms-site/src/previewskeleton/PreviewSkeleton.sc.ts

```diff
@@ -37,10 +37,32 @@ export const RowSkeleton = styled.div<RowsSkeletonStyleProps>`
 interface ImageSkeletonStyleProps {
     $backgroundColor: string;
     $color: string;
+    $aspectRatio: string | number | undefined;
+    $height: string | number | undefined;
 }
+
 export const ImageContainer = styled.div<ImageSkeletonStyleProps>`
     background-color: ${({ $backgroundColor }) => $backgroundColor};
     color: ${({ $color }) => $color};
     width: 100%;
-    height: 300px;
+
+    ${({ $aspectRatio }) =>
+        typeof $aspectRatio !== "undefined" &&
+        css`
+            aspect-ratio: ${$aspectRatio};
+        `}
+
+    ${({ $aspectRatio, $height }) =>
+        typeof $aspectRatio === "undefined" &&
+        typeof $height !== "undefined" &&
+        css`
+            height: ${$height};
+        `}
+
+    ${({ $aspectRatio, $height }) =>
+        typeof $aspectRatio === "undefined" &&
+        typeof $height === "undefined" &&
+        css`
+            height: 300px;
+        `}
```

Comment:
```suggestion
    ${({ $aspectRatio, $height = "300px" }) =>
        typeof $aspectRatio !== "undefined"
            ? css`
                  aspect-ratio: ${$aspectRatio};
              `
            : css`
                  height: ${$height};
              `}
```

Alternatively, you could use this.

Reply:
Ah, true 😄 

Done in 56efdd8cc8acaf6a704ded8c083a7ec9219384d5